bfe.json

```diff
@@ -3,28 +3,31 @@
     "code": 0,
     "type": "feed",
     "formats": [
-      { "code": 0, "format": "classic",      "data_length": 32, "sigil": "@", "suffix": ".ed25519" },
-      { "code": 1, "format": "gabbygrove-v1",  "data_length": 32 },
-      { "code": 2, "format": "bamboo",       "data_length": 32 },
-      { "code": 3, "format": "bendybutt-v1", "data_length": 32 }
+      { "code": 0, "format": "classic",       "data_length": 32, "sigil": "@", "suffix": ".ed25519" },
+      { "code": 1, "format": "gabbygrove-v1", "data_length": 32 },
+      { "code": 2, "format": "bamboo",        "data_length": 32 },
+      { "code": 3, "format": "bendybutt-v1",  "data_length": 32 },
+      { "code": 4, "format": "buttwoo-v1",      "data_length": 32 }
     ]
   },
   {
     "code": 1,
     "type": "message",
     "formats": [
       { "code": 0, "format": "classic",      "data_length": 32, "sigil": "%", "suffix": ".sha256" },
-      { "code": 1, "format": "gabbygrove-v1",  "data_length": 32 },
+      { "code": 1, "format": "gabbygrove-v1","data_length": 32 },
       { "code": 2, "format": "cloaked",      "data_length": 32, "sigil": "%", "suffix": ".cloaked" },
       { "code": 3, "format": "bamboo",       "data_length": 64 },
-      { "code": 4, "format": "bendybutt-v1", "data_length": 32 }
+      { "code": 4, "format": "bendybutt-v1", "data_length": 32 },
+      { "code": 5, "format": "buttwoo-v1",     "data_length": 32 }
     ]
   },
   {
     "code": 2,
     "type": "blob",
     "formats": [
-      { "code": 0, "format": "classic", "data_length": 32, "sigil": "&", "suffix": ".sha256" }
+      { "code": 0, "format": "classic", "data_length": 32, "sigil": "&", "suffix": ".sha256" },
+      { "code": 1, "format": "buttwoo-v1", "data_length": 32, "sigil": "&", "suffix": ".butt2" }
```

Comment:
Is the suffix gonna be butt2? And what are buttwoo blobs?

Reply:
Right, no need to add that (yet). I was thinking that if we have blake3 we might as well do blobs using that hash function instead. But first lets get the format in. Good spotting.